### PR TITLE
Develop Sidebar

### PR DESCRIPTION
**Pull request by Dev-SheikhSuhail**
Sidebar-Desktop2

Sidebar-Tablet2
Sidebar-Mobile2

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+body {
+  background: rgb(241, 245, 248);
```
Comment (UmmarHamid) [inline styles.css]: Can we add a subtle color here?

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+.sidebar {
+  width: 30%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background: rgba(139, 94, 94, 0.16);
+  visibility: hidden;
+}
+.show-sidebar {
+  visibility: visible;
```
Comment (UmmarHamid) [inline styles.css]: Are we able to click through this when it is not visible? We should be.
Using display none would not allow transitions to work
Consider toggling width from 0 to current size on show sidebar.

File: styles.css (hunk 1 above)
Comment (Dev-SheikhSuhail) [reply styles.css]: Of course.

File: styles.css (hunk 2 above)
Comment (Dev-SheikhSuhail) [reply styles.css]: Definitely, I'll resolve it.
I didn't consider it, as our webpage body didn't have any other components on the left side. That is why, I didn't check it out.
Thanks for your suggestion.

**Comment by UmmarHamid** [comment]
@Dev-SheikhSuhail I cannot find the updated screenshots, can you help me with that?

**Comment by Dev-SheikhSuhail** [comment]
Added new Screenshots.

**Pull request merged**